ticketutil/servicenow.py

```diff
@@ -0,0 +1,459 @@
+import os
+import logging
+import requests
+import json
+
+from ticketutil.ticket import Ticket
+
+__author__ = 'dranck, rnester, kshirsal, pzubaty'
+
+STATE = {'new':'0',
```

Comment:
well, in the end I have replaced `STATE` with `self.available_states` which is being set up in `_verify_project()` This will be in new commit which is going to be pushed in 30 minutes.